docs/developer_guide.md

```diff
@@ -151,33 +150,30 @@ public void TestCleanup()
     TestEnvironment.AssemblyCleanup(TestType.Nuget);
 }
 ```
-The test apps are using released versions of MUX NuGet package locally. In [CI](https://dev.azure.com/ms/microsoft-ui-xaml/_build?definitionId=20), 
+The test apps are using released versions of the WinUI NuGet package locally. In [CI](https://dev.azure.com/ms/microsoft-ui-xaml/_build?definitionId=20), 
 the test pipeline will generate a NuGet package for each build, and there's a 
-separate pipeline configured to consume the generated package from latest 
+separate pipeline configured to consume the generated package from the latest 
 build and run MUXControl.ReleaseTest.
 
 #### Downlevel testing
 
-One of the core values of WinUI is the way that it brings controls to a wide 
-variety of versions of Windows, handling version compatibility differences so 
-that developers using WinUI don't have to. As such, testing WinUI changes on 
-different versions is sometimes necessary. To accomplish this you will need to 
-at times set up older versions of Windows for testing. To get these earlier 
-versions you can make use of a Visual Studio subscription [as described here](https://docs.microsoft.com/azure/virtual-machines/windows/client-images).
+One of the core features of WinUI is that it maintains down-level compatibility with earlier versions of Windows 10, handling version compatibility differences so 
+that developers using WinUI don't have to. As such, testing your changes on 
+different Windows versions is sometimes necessary. Windows versions can be acquired using a Visual Studio subscription [as described here](https://docs.microsoft.com/azure/virtual-machines/windows/client-images), if required.
```

Comment:
@ranjeshj Are there ways to get a VM if you don't have any of the subscriptions listed here?

Reply:
I don't think so :(

---
In reply to: [485197625](https://github.com/microsoft/microsoft-ui-xaml/pull/3262#discussion_r485197625) [](ancestors = 485197625)